packages/flutter/test/painting/text_span_test.dart

```diff
@@ -44,7 +43,6 @@ void main() {
             TextSpan(),
           ],
         ),
-        null,
```

Comment:
Seems like this code isn't correct, since TextSpan is not documented as accepting null children